src/app/pages/items/items.component.html

```diff
@@ -1,16 +1,15 @@
 <section>
```

Comment:
No h1 in the page, (and no h2 neither, if I compare your member cards you have h3. thus you have to implement h1 and h2 before